lib/apipie_bindings/action.rb

```diff
@@ -48,7 +48,7 @@ def find_route(params={})
       sorted_routes = routes.sort_by { |r| [-1 * r.params_in_path.count, r.path] }
 
       suitable_route = sorted_routes.find do |route|
-        route.params_in_path.all? { |path_param| params.keys.map(&:to_s).include?(path_param) }
+        route.params_in_path.all? { |path_param| params.keys.map(&:to_s).include?(path_param) && (!params[path_param.to_sym].nil? || !params[path_param].nil?) }
```

Comment:
Do we still need the first part of the condition when we're checking for nil anyway? I think it could be simplified just to `!params[path_param.to_sym].nil? || !params[path_param].nil?`

Reply:
Indeed, you are right. Non-nil path_param implies it is included in request params. Will update it.